utils/scripts/helpers/install_maldet.sh

```diff
@@ -33,7 +33,7 @@ mkdir -p /etc/clamd.d
 printf "LogSyslog yes\n" >> /etc/clamd.d/scan.conf
 
 lmd_tarball='maldetect-current.tar.gz'
-: "${REPO_FILE_URL:=https://artifactory.dc.hpdd.intel.com/artifactory/repo-files/}"
+: "${REPO_FILE_URL:=https://artifactory/artifactory/repo-files/}"
```

Comment:
Should this be using ${ARTIFACTORY_URL}?

Reply:
No.   Different functionality.   This is a complete URL to download pre-built repository files that is totally independent of what type of server is serving the files or hosting the repositories.